pandas/io/pytables.py

```diff
@@ -1099,7 +1091,7 @@ def put(
             <https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#query-via-data-columns>`__.
         encoding : str, default None
             Provide an encoding for strings.
-        dropna   : bool, default False, do not write an ALL nan row to
+        dropna : bool, default False, do not write an ALL nan row to
```

Comment:
new line after "default False"

Reply:
It turns out that this parameter is not even used in the function. Deleted the corresponding strings.